atom_core/src/atom_core/dataset_io.py

```diff
@@ -858,14 +898,46 @@ def addNoiseToInitialGuess(dataset, args, selected_collection_key):
         if sensor_key != dataset['calibration_config']['anchored_sensor']:
             calibration_child = sensor['calibration_child']
             calibration_parent = sensor['calibration_parent']
-            addNoiseToTF(dataset, selected_collection_key, calibration_parent,
-                         calibration_child, nig_trans, nig_rot)
 
+            tf_to_add_noise = generateKey(calibration_parent,calibration_child)
+            if tf_to_add_noise in ntfl_tfs:
+                atomWarn(f'Not adding initial guess noise to {tf_to_add_noise} because its defined in -ntfl')
+                continue
+
+            addNoiseToTF(dataset, selected_collection_key, calibration_parent, calibration_child, nig_trans, nig_rot)
 
-def addNoiseToTF(dataset, selected_collection_key, calibration_parent, calibration_child, nig_trans, nig_rot):
+# TODO make a basic function called by fixed and multiple
 
+def computeNoise(initial_translation,initial_euler_angles,translation_noise_magnitude,rotation_noise_magnitude):
+    '''
+    Computes both the translation and rotation noise given a certain magnitude and returns the new noisy translation/rotation vectors
+    '''
+    # Translation
+
+    v = np.random.uniform(-1.0, 1.0, 3)
+    v = v / np.linalg.norm(v)
+    translation_delta = v * translation_noise_magnitude
+
+    new_translation = initial_translation + translation_delta
+    # Rotation
+
+    # Its necessary to redefine 'v' to ensure that the translation and rotation noise aren't always the same given the same magnitude
+    v = np.random.uniform(-1.0, 1.0, 3)
+    v = v / np.linalg.norm(v)
+    rotation_delta = v * rotation_noise_magnitude
+
+    new_euler_angles = initial_euler_angles + rotation_delta
+
+    return new_translation,new_euler_angles
+
+
+def addNoiseToTF(dataset, selected_collection_key, calibration_parent, calibration_child, noise_trans, noise_rot):
+
+    print(Fore.RED + 'Transformation parent ' + calibration_parent + ' child ' + calibration_child + Style.RESET_ALL)
     transform_key = generateKey(calibration_parent, calibration_child, suffix='')
 
+    atomWarn(f'Adding noise bigger than 1.0m/rad right now is bugged and might yield unexpected results. Check issue #929 for more information')
```

Comment:
After reading the issues, I think I agree with @miguelriemoliveira in #929. This should be an error thrown when `noise_trans` or `noise_rot` is greater than 1 instead of just a warning printed everytime noise is added.

This string doesn't need to be an f-string and besides, in terms of code uniformity, ATOM doesn't really use f-strings as a norm. It's not that important but it would be nice to have consistency in the code's format, I think.

In terms of the message itself, I'd change "right now" to "at the moment"; it's cleaner.